golos.publication/golos.publication.cpp

```diff
@@ -834,9 +834,12 @@ void publication::set_params(std::vector<posting_params> params) {
     param_helper::set_parameters<posting_params_setter>(params, cfg, _self);
 }
 
-void publication::reblog(name rebloger, structures::mssgid message_id) {
+void publication::reblog(name rebloger, structures::mssgid message_id, std::string headermssg, std::string bodymssg) {
     require_auth(rebloger);
 
+    eosio_assert(headermssg.length() < config::max_length, "Title length is more than 256.");
+    eosio_assert(bodymssg.length(), "Body is empty.");
+
```

Comment:
in Golos we have "You cannot reblog your own content" assert. propose to disallow author to reblog his own posts.
Also Golos reblog disables empty body if title set: "Body is empty but Title or JSON Metadata is set"